Dockerfile.dev

```diff
@@ -9,13 +9,6 @@ RUN npm install
 
 COPY . .
 
-# Stage 2: Final image
```

Comment:
What is the current image size? As you removed the multistage building dev docker

Reply:
It's around 1.5 gigs but I removed the multi stage build because they were causing a lot of trouble because we are using dev environment in images not pre built js files. So there is no point of multi stage builds until we have our build process sorted.

Reply:
Sounds Good! We need to sort out the build process before it becomes complex

Reply:
But we are not using any built js files in the dev docker as we using an entire directory with a smaller size image in the final stage. I suggest keeping it (The image size is reduced by ~1GB if we have muti-stage builds)

Reply:
It's not possible because node alpine/slim (images we were using in multi stage builds) will always show many abnormalities and bugs due to version incompatibility

Reply:
We can't remove any additional files as the image is running in dev mode not production mode